src/main/java/com/exadel/discount/service/VendorLocationService.java

```diff
@@ -0,0 +1,19 @@
+package com.exadel.discount.service;
+
+import com.exadel.discount.dto.location.CreateLocationDTO;
+import com.exadel.discount.dto.location.LocationDTO;
+import com.exadel.discount.entity.VendorLocation;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface VendorLocationService {
+
+    LocationDTO save(CreateLocationDTO createLocationDTO);
+
+    LocationDTO getById(UUID id);
+
+    List<LocationDTO> getAll(UUID vendorId);
```

Comment:
please rename method. 
if there is parameter please put it on method name like
getLocationsByVendorId or getLocationsForVendorById, etc.

Reply:
+